### PR TITLE
Add release notes for 2.15

### PR DESCRIPTION
**Pull request by rbhavna**
### Description
add release notes for 2.15
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by jngz-es — APPROVED** [review]

File: release-notes/opensearch-ml-common.release-notes-2.15.0.0.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Version 2.15.0.0 Release Notes
+
+Compatible with OpenSearch 2.15.0.0
+
+### Enhancements
+* add a connector blueprint for Amazon Comprehend APIs ([#2470](https://github.com/opensearch-project/ml-commons/pull/2470))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.15.0.0.md:6]: This is Documentation

File: release-notes/opensearch-ml-common.release-notes-2.15.0.0.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* add titan embeeding v2 to blueprint ([#2480](https://github.com/opensearch-project/ml-commons/pull/2480))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.15.0.0.md:7]: move to Documentation

File: release-notes/opensearch-ml-common.release-notes-2.15.0.0.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* Implementing retry for remote connector to mitigate throttling issue ([#2462](https://github.com/opensearch-project/ml-commons/pull/2462))
+* Add connector tool ([#2516](https://github.com/opensearch-project/ml-commons/pull/2516))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.15.0.0.md:9]: Should be a Feature

File: release-notes/opensearch-ml-common.release-notes-2.15.0.0.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* guardrails model support ([#2491](https://github.com/opensearch-project/ml-commons/pull/2491))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.15.0.0.md:10]: should be a Feature

File: release-notes/opensearch-ml-common.release-notes-2.15.0.0.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* ml inference ingest processor support for local models ([#2508](https://github.com/opensearch-project/ml-commons/pull/2508))
+* add setting to allow private IP ([#2534](https://github.com/opensearch-project/ml-commons/pull/2534))
+
+
+### Bug Fixes
+* hanlde the throttling error in the response header ([#2442](https://github.com/opensearch-project/ml-commons/pull/2442))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.15.0.0.md:16]: move to Enhancements

**Comment by ylwu-amzn** [comment]
Check if any missing PR ?

**Review by jngz-es — APPROVED** [review]

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**